docs/plans/2026-05-10-rd-07-desktop-videoplayer-adapter-plan.md

```
@@ -0,0 +1,849 @@
+**Plan Status:** active
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)